icc-x-api/icc-crypto-x-api.ts

```diff
@@ -412,10 +455,12 @@ export class IccCryptoXApi {
           )
 
           const allCryptedForeignKeys = _.cloneDeep(modifiedObject.cryptedForeignKeys || {})
-          allCryptedForeignKeys[delegateId] = _.uniqBy(
-            cryptedForeignKeysCryptedDecrypted,
-            (x: any) => x.k
-          ).map((x: any) => x.d)
+          if (cryptedForeignKeysCryptedDecrypted.length > 0) {
```

Comment:
@cebamps Does this solve the `parentId` being an empty array like we noticed with @adrian-meyvn ?

Reply:
Seems so. I added a reference to that commit in our ticket for that (MS-1182)

Reply:
https://medispring.atlassian.net/browse/MS-1182